cmd/tiller-proxy/main.go

```diff
@@ -181,6 +181,16 @@ func main() {
 		negroni.Wrap(handlerutil.WithParams(h.DeleteRelease)),
 	))
 
+	// Backend routes unrelated to tiller-proxy functionality.
+	// TODO(mnelson): Once the helm3 support is complete and tiller-proxy is being removed,
+	// reconsider where these endpoints live.
+	appreposHandler := handler.AppRepositories{}
+	backendAPIv1 := r.PathPrefix("/backend/v1").Subrouter()
+	backendAPIv1.Methods("POST").Path("/apprepositories").Handler(negroni.New(
+		authGate,
```

Comment:
I've used `authGate` here simply because it appears for all the other handlers, but I don't see why it's necessary here, as I'll be hitting the k8s API with the users credentials (with the `id_token` or `access_token` sent with the request). I assume it's safe to remove here?

Reply:
yes, it's probably not needed. I'd only see it useful if we do something (e.g. fetching credentials) before doing the actual request to the k8s API.